benchmark/scenarios/_request_load.py

```diff
@@ -133,15 +130,15 @@ def handle_request_error(result):
         if count != 1:
             Message.log(function='_request_and_measure', count=count)
 
-        for i in range(count):
+        for _ in range(count):
             self.rate_measurer.update_rate()
 
-            for i in range(self.request_rate):
+            for _ in range(self.request_rate):
                 t0 = self.reactor.seconds()
 
                 d = self.request.make_request()
 
-                def get_time(_ignore, reactor=self.reactor, t0=t0):
+                def get_time(_, reactor=self.reactor, t0=t0):
```

Comment:
:question: With a more flexible `dummy-variables-rgx` in `.pylintrc` this variable should be ignored.